Developers/Newcomers_Guide.rst

```diff
@@ -9,10 +9,12 @@ to complete. The newcomer onboarding process is as follows:
 You will start as a newcomer, which is kind of a trial. If you complete the
 following tasks, you will become a developer at coala:
 
-  - merge a ``difficulty/newcomer`` Pull Request
-  - review at least a ``difficulty/newcomer`` Pull Request
-  - merge a ``difficulty/low`` Pull Request
-  - review at least a``difficulty/low`` or higher Pull Request
+  - solve a ``difficulty/newcomer`` issue
+  - open a pull request for your issue and get it merged
+  - review at least one ``difficulty/newcomer`` Pull Request
+  - solve a ``difficulty/low`` issue
+  - open a pull request for your issue and get it merged
```

Comment:
oh I was shortly confused by this. My mind was like "I know this line", maybe it can be easier to grasp if we tell them like "there's two steps, first newcomer, ..., then low, ..."?